lib/gather-telemetry.js

```diff
@@ -0,0 +1,250 @@
+const puppeteer = require("puppeteer");
+const getRepoInfo = require("git-repo-info");
+const Cache = require("sync-disk-cache");
+
+const gitInfo = getRepoInfo();
+const cache = new Cache(`native-class-codemod-${gitInfo.sha}`);
```

Comment:
Can we also add the current working directory to the cache key? Just using the SHA _can_ collide (yes, I know its probably silly, but it seems easy enough to add `process.cwd()` here...)